server/src/api/v1/users/auth-user.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from "express";
+import { wrapResponse } from "../../../functions/response-wrapper";
+import { IncomingUser } from "../../../interfaces/users.interface";
+import { User } from "../../../models/user.model";
+const jwt = require("jsonwebtoken"); //TODO make import statement
```

Comment:
Typings dazu installieren npm i -s @types/jsonwebtoken (installiert https://www.npmjs.com/package/@types/jsonwebtoken)

Dann kannst du dein import auch machen :) 
```suggestion
import { jwt } from 'jsonwebtoken';
```